### PR TITLE
alt-tab alarm, not verified for mobile (I love you<3)

### PR DESCRIPTION
**Pull request by Frogz9000**

### CODE REVIEW

File: client/src/index.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { GameManager, createGameManager } from './game/gameManager';
 import StartMenu from './menu/main/StartMenu';
 import { Player } from './game/gameState.d';
 import LoadingScreen from './menu/LoadingScreen';
+import { collapseTextChangeRangesAcrossMultipleVersions } from 'typescript';
```
Comment (Jack-Papel) [inline client/src/index.tsx:9]: Unused import
```suggestion
```

File: client/src/index.tsx [hunk 2]
```diff
@@ -90,4 +91,12 @@ export function replaceMentions(rawText: string, players: Player[]) {
...
\ No newline at end of file
+}
+
+window.addEventListener("blur", () => {
+    Anchor.onBlur();
+});
+
+window.addEventListener("focus", ()=>{
+    Anchor.onFocus();
+})
```
Comment (Jack-Papel) [inline client/src/index.tsx:102]: Should go in `Anchor` `onComponentMount` function

Comment (Jack-Papel) [reply client/src/index.tsx:102]: Or inline these Anchor.onBlur and Anchor.onFocus functions.

**Review by Jack-Papel — CHANGES_REQUESTED** [review]

**Comment by Frogz9000** [comment]
This should be made into a pull request draft :)

**Comment by Jack-Papel** [comment]
Stale

**Pull request closed without merging**